### PR TITLE
Update Annotations versions

### PR DESCRIPTION
**Pull request by Majkl578**

### CODE REVIEW

**Comment by jwage** [comment]
Thank you sir

**Comment by jwage** [comment]
@Majkl578 branch named 1.6 was never created for annotations

File: app/config/projects.yml [hunk 1]
```diff
@@ -286,10 +286,22 @@ parameters:
...
                     slug: 'latest'
+                    upcoming: true
+                -
+                    name: '1.7'
+                    branchName: '1.7'
+                    slug: '1.7'
+                    upcoming: true
+                -
+                    name: '1.6'
+                    branchName: '1.6'
```
Comment (jwage) [inline app/config/projects.yml]: Branch named 1.6 was never created for annotations.

Comment (Majkl578) [reply app/config/projects.yml]: Ah, there is 1.5 and 1.7. 😂 Created from v1.6.0 tag.

**Pull request merged**